pkgs/by-name/co/copilot-language-server/package.nix

```diff
@@ -24,8 +25,23 @@ let
     }
     ."${stdenvNoCC.hostPlatform.system}"
       or (throw "Unsupported system: ${stdenvNoCC.hostPlatform.system}");
-in
 
+  executableName = "copilot-language-server";
+  fhs =
+    { package }:
+    buildFHSEnv {
+      name = package.meta.mainProgram;
+      version = package.version;
+      targetPkgs = pkgs: [ pkgs.stdenv.cc.cc.lib ];
+      runScript = lib.getExe package;
+
+      meta = package.meta // {
+        description =
+          package.meta.description
+          + " (FHS-wrapped). Use this version if you have trouble with the normal one.";
```

Comment:
Shouldn't we mention here what kind of trouble like that executables cannot be executed?

Reply:
Could I reference the issue #391730 in the text? And if so, as link or just the issue number?

Reply:
A comment in the form of ``Could not start dynamically linked executable`` plus a link to that issue would be super cool :)